projects/packages/forms/src/dashboard/integrations/index.tsx

```diff
@@ -32,7 +32,8 @@ const Integrations = () => {
 		mailpoet: false,
 	} );
 
-	const isMailpoetEnabled = config( 'isMailpoetEnabled' );
+	const config = useFormsConfig();
+	const isMailpoetEnabled = Boolean( config?.isMailPoetEnabled );
```

Comment:
An example of replacing the config object/method with the new approach for getting data.